### DISCUSSION TITLE
ECIP Editor Updating

### DISCUSSION THREAD

**Discussion by stevanlohja** [ethereumclassic/ECIPs#448] [ECIPs]
Discussion for updating ECIP Editors.

**Comment by stevanlohja** [comment]
Current ECIP editors are listed in [ECIP-1000](https://github.com/ethereumclassic/ECIPs/blob/master/_specs/ecip-1000.md):

@meowsbits ✔️
@soc1c ✔️
~~@BelfordZ~~
~~@faraggi~~
@gitr0n1n ✔️
~~@stevanlohja~~

- **Can the above members confirm they are continuing to be ECIP Editors?**
- **New ECIP Editors are welcomed, introduce yourself!** Ideally, you should have a basic understanding of EIP/ ECIPs, Git/ Github, and it's really just a neutral-volunteer role to simply QA ECIPs that come through and nurture the ECIP repo.

**Reply by stevanlohja** [reply]
~~@stevanlohja~~ Removing myself for now ✌️

**Comment by meowsbits** [comment]
Confirm.

**Comment by faraggi** [comment]
I'm going to have to step down, feel free to remove me from the list.

**Comment by stevanlohja** [comment]
Pinging @soc1c @BelfordZ @gitr0n1n 
I've DMed everyone but gitr0n1n so if you have success contacting him please do!

**Comment by gitr0n1n** [comment]
👋

**Comment by soc1c** [comment]
Confirm

**Comment by stevanlohja** [comment]
Antoine says he'll volunteer

**Comment by IstoraMandiri** [comment]
I can help with this
